.github/workflows/lotus-devnet-publish.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           context: ./scripts/devnet
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          push: ${{ github.ref == 'refs/heads/main' }}
+          push: true
```

Comment:
Push even in PRs?

Reply:
This will work only on workflow dispatch. This way, you can release a new tagged Lotus release (like the one in this PR) without spamming `main` with Lotus-Devnet upgrades. https://github.com/ChainSafe/forest/pkgs/container/lotus-devnet